### PR TITLE
feat: wallet installation

### PR DESCRIPTION
**Pull request by miliPaoletti**
Add set up account step for copilot.

### CODE REVIEW

**Comment by sandoche** [comment]
@miliPaoletti the deploy preview broke because of this error:
```
10:02:00 AM: mission-page:build: cache bypass, force executing ae3f8dba21b9ddb4
10:02:00 AM: mission-page:build: $ sh setup-redirects.sh && next build
10:02:00 AM: mission-page:build: warn  - No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
10:02:00 AM: mission-page:build: info  - Linting and checking validity of types...
10:02:04 AM: mission-page:build: Failed to compile.
10:02:04 AM: mission-page:build: 
10:02:04 AM: mission-page:build: ./src/components/mission/TopBarMissionControl.tsx:10:8
10:02:04 AM: mission-page:build: Type error: Cannot find module 'helpers' or its corresponding type declarations.
10:02:04 AM: mission-page:build: 
10:02:04 AM: mission-page:build:    8 |   amountToDollars,
10:02:04 AM: mission-page:build:    9 |   displayTopBarTooltip,
10:02:04 AM: mission-page:build: > 10 | } from helpers;
10:02:04 AM: mission-page:build:      |        ^
10:02:04 AM: mission-page:build:   11 | 
10:02:04 AM: mission-page:build:   12 | import { useHeaderInfo } from ../../internal/functionality/hooks/useHeaderInfo;
10:02:04 AM: mission-page:build:   13 | import useAssetsTopBar from ../../internal/functionality/hooks/useAssetsTopBar;
10:02:04 AM: mission-page:build: error Command failed with exit code 1. (https://ntl.fyi/exit-code-1)
10:02:04 AM: mission-page:build: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
10:02:04 AM: mission-page:build: ERROR: command finished with error: command (/opt/build/repo/apps/mission) yarn run 
```

It's weird because your turbo.json have the right settings it seems:
```
    "mission-page#build": {
      "dependsOn": ["evmos-wallet#build", "helpers#build", "services#build"],
      "cache": false
    },
```

**Comment by sandoche** [comment]
If we don't find a solution to the broken build I guess we will have to revert this commit (and change the settings in Netlify):
https://github.com/evmos/apps/pull/384

**Review by sandoche — APPROVED** [review]
It's good there are just a few little changes.

Also the deploy preview broke (this is linked to my optimization) so I could't try.

File: apps/mission/src/components/copilot/steps/buttons/ButtonCopilot.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  name: string;
+  index: number;
+  stepsLength: number;
+  status: string;
+  handleClick: () => Promise<void>;
+  textError: string;
+  statusButton: string;
+};
+
+const STYLE1 = {
```
[the remark this answers is not in the document]
Comment (miliPaoletti) [reply apps/mission/src/components/copilot/steps/buttons/ButtonCopilot.tsx:19]: Those styles are for the stepper. So the first style is for the vertical line. The second one is for the circles of the stepper and the third one is for the button

**Pull request merged**